corehq/apps/app_manager/templates/app_manager/v1/form_view_base.html

```diff
@@ -118,6 +118,7 @@
             return result;
         }
 
+        var form_requires = ko.observable({{ form.requires|JSON }});
```

Comment:
It's used in `FormFilter` a bit below this, all I'm doing is moving it closer to where it's used. The `{{ form.requires|JSON }}` here is indeed equivalent to using `initial_page_data('form_requires')`. No particular reason for not using initial_page_data here, probably just copied/pasted before I added the initial_page_data code. I'll come back to this eventually, planning on getting this whole file externalized over the next week or two.